### ISSUE TITLE
Doubt in use of labels in cross_validation_accuracy

### ISSUE THREAD

**Issue by SanchayanMaity** [iit-cs579/main#425]
In lec11 jupyter notebook, cross_validation_accuracy is defined as

```
# 5-fold cross-validation accuracy
from sklearn.model_selection import KFold

def do_cross_validation(model, X, y, n_folds):
    cv = KFold(n_splits=n_folds, shuffle=True, random_state=42)
    accuracies = []
    for train_ind, test_ind in cv.split(X):
        model.fit(X[train_ind], y[train_ind])
        predictions = model.predict(X[test_ind])
        accuracies.append(accuracy(y[test_ind], predictions))
```
Do I understand correctly that 'labels' is to passed in place of y. My results do not match with Log.txt and I suspect the tokenizer or cross_validation_accuracy implementation is wrong.
